containerize/Containerfile

```diff
@@ -5,34 +5,61 @@
 # Prepare dependencies
 FROM registry.access.redhat.com/ubi9-minimal AS deps
 
+# Indicate if dependencies were prefetched using Cachi2
+# They mush be located at /cachi2/output/deps
+ARG PREFETCH=false
+
+# These versions should be consistent with the listed in the artifacts.lock.yaml file
+ARG ZAP_VERSION=2.15.0
+ARG FF_VERSION=128.6.0esr
+ARG K8S_VERSION=1.32.1
+ARG TRIVY_VERSION=0.49.1
```

Comment:
Trivy version now should be 0.59.0: https://github.com/RedHatProductSecurity/rapidast/commit/afc218b54f98aa915f083eaa7d777ae915718b2d

Reply:
Done